torch/distributed/_tensor/api.py

```diff
@@ -402,13 +403,17 @@ def distribute_tensor(
             first rank of each dimension of the `device_mesh`.
 
     Returns:
-        A :class:`DTensor` object
+        A :class:`DTensor(Tensor)` or `XLAShardedTensor(Tensor)` object
```

Comment:
nit: why there's a "(Tensor)" type there? I think we can just say
```
A :class:`DTensor` or `XLAShardedTensor` object
```

Reply:
Done